templates/web/surrey/footer_extra.html

```diff
@@ -0,0 +1,106 @@
+                        <a class="scc-footer__link" href="https://www.surreycc.gov.uk/site-map">Site map</a>
+                    </li>
+                </ul>
+            </div>
+
+
+        </div>
+
+        <div class="row">
+            <div class="col-12">
+                <div class="scc-social-icons">
+                    <ul class="scc-social-icons__list mb-3">
+                        <li class="scc-social-icons__item">
+                            <a class="scc-social-icons__link scc-footer__link scc-footer__link--icon fa-brands fa-x-twitter" href="https://twitter.com/SurreyNews">
+                                <span class="sr-only">Follow us on Twitter</span>
+                            </a>
+                        </li>
+                        <li class="scc-social-icons__item">
+                            <a class="scc-social-icons__link scc-footer__link scc-footer__link--icon fa-brands fa-facebook" href="https://www.facebook.com/Surrey-County-Council-126261804063269/">
+                                <span class="sr-only">Follow us on Facebook</span>
+                            </a>
+                        </li>
+                        <li class="scc-social-icons__item">
+                            <a class="scc-social-icons__link scc-footer__link scc-footer__link--icon fa-brands fa-youtube" href="https://www.youtube.com/user/SurreyCountyCouncil">
+                                <span class="sr-only">Follow us on YouTube</span>
+                            </a>
+                        </li>
+                        <li class="scc-social-icons__item">
+                            <a class="scc-social-icons__link scc-footer__link scc-footer__link--icon fa-brands fa-instagram" href="https://www.instagram.com/surreycountycouncil/">
+                                <span class="sr-only">Follow us on Instagram</span>
+                            </a>
+                        </li>
+                        <li class="scc-social-icons__item">
+                            <a class="scc-social-icons__link scc-footer__link scc-footer__link--icon fa-brands fa-linkedin" href="https://www.linkedin.com/company/surrey-county-council/careers?trk=top_nav_careers">
+                                <span class="sr-only">Follow us on LinkedIn</span>
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-12">
+                <ul class="scc-footer__list">
+                    <li class="scc-footer__item">
+                        <a class="scc-footer__link" href="https://www.surreycc.gov.uk/"><span class="sr-only">Surrey County Council </span>Home</a>
+                    </li>
+                    <li class="scc-footer__item">
+                        © Surrey County Council 2024
+                    </li>
+                    
+                        <li class="scc-footer__item">
```

Comment:
I notice this exists on their own site, but does it make sense for it to be included on FMS?